mentions/src/main/kotlin/io/spine/examples/pingh/mentions/GitHubClientProcess.kt

```diff
@@ -142,4 +129,30 @@ public class GitHubClientProcess :
     internal fun inject(gitHubClientService: GitHubClientService) {
         this.gitHubClientService = gitHubClientService
     }
+
+    private companion object {
+        /**
+         * Determines and returns the time from which mentions should be updated.
+         *
+         * If updates were already performed, it returns the time of the last successfully
+         * completed update. If no updates are made, it returns the time equal to midnight
+         * of the previous work day.
+         */
+        private fun determineFromWhenFetchMentions(whenLastSuccessfulUpdate: Timestamp): Timestamp {
```

Comment:
How about turning this into a private extension to `Timestamp`? E.g.

`someTimestamp.thisOrLastWorkday()`?